Content/GUI/Armor/UIGearInventory.cs

```diff
@@ -0,0 +1,897 @@
+			: LoadoutPadding + Main.screenWidth;
+
+		float top = Main.mapStyle == 1 ? 446f : 190f;
+
+		root.Left.Set(MathHelper.SmoothStep(root.Left.Pixels, left, Smoothness), 0f);
+		root.Top.Set(MathHelper.SmoothStep(root.Top.Pixels, top, Smoothness), 0f);
+	}
+
+	private void HandleLeftPageClick(UIMouseEvent @event, UIElement element)
+	{
+		SoundEngine.PlaySound(
+			SoundID.MenuTick with
+			{
+				Pitch = 0.25f,
+				MaxInstances = 1
+			}
+		);
+
+		CurrentPage--;
+	}
+
+	private void HandleRightPageClick(UIMouseEvent @event, UIElement element)
+	{
+		SoundEngine.PlaySound(
+			SoundID.MenuTick with
+			{
+				Pitch = 0.25f,
+				MaxInstances = 1
+			}
+		);
+
+		CurrentPage++;
+	}
+
+	private static void UpdateLoadoutIcon(UIImage image, int index)
+	{
+		if (Player.CurrentLoadoutIndex == index)
+		{
+			image.Color = Color.White;
+			image.Left.Pixels = MathHelper.SmoothStep(image.Left.Pixels, -LoadoutPadding, Smoothness);
+		}
+		else
+		{
+			image.Left.Pixels = MathHelper.SmoothStep(image.Left.Pixels, 0f, Smoothness);
+			image.Color = Color.DarkGray;
+		}
+	}
+
+	private static void UpdateMouseOver(UIMouseEvent @event, UIElement element)
+	{
+		SoundEngine.PlaySound(
+			SoundID.MenuTick with
+			{
+				Pitch = 0.15f,
+				MaxInstances = 1
+			}
+		);
+	}
+
+	private static void UpdateMouseOut(UIMouseEvent @event, UIElement element)
+	{
+		SoundEngine.PlaySound(
+			SoundID.MenuTick with
+			{
+				Pitch = -0.25f,
+				MaxInstances = 1
+			}
+		);
+	}
+
+	private static UIElement BuildDefaultInventory()
+	{
+		var inventoryRoot = new UIElement
+		{
+			Width = StyleDimension.FromPixels(ArmorPageWidth),
+			Height = StyleDimension.FromPixels(ArmorPageHeight)
+		};
+
+		var wings = new UIHoverImageItemSlot(DefaultFrameTexture, WingsIconTexture, ref Player.armor, 4, ItemSlot.Context.EquipAccessory)
+		{
+			ActiveScale = 1.15f,
+			ActiveRotation = MathHelper.ToRadians(1f)
+		};
+
+		wings.OnMouseOver += UpdateMouseOver;
+		wings.OnMouseOut += UpdateMouseOut;
+
+		wings.Predicate = (item, _) => item.wingSlot > 0;
+
+		inventoryRoot.Append(wings);
+
+		var helmet = new UIHoverImageItemSlot(DefaultFrameTexture, HelmetIconTexture, ref Player.armor, 0, ItemSlot.Context.EquipArmor)
+		{
+			HAlign = 0.5f,
+			VAlign = 0f,
+			ActiveScale = 1.15f,
+			ActiveRotation = MathHelper.ToRadians(1f)
+		};
+
+		helmet.OnMouseOver += UpdateMouseOver;
+		helmet.OnMouseOut += UpdateMouseOut;
+
+		helmet.Predicate = (item, _) => item.headSlot > 0;
+
+		inventoryRoot.Append(helmet);
+
+		var necklace = new UIHoverImageItemSlot(DefaultFrameTexture, NecklaceIconTexture, ref Player.armor, 5, ItemSlot.Context.EquipAccessory)
+		{
+			HAlign = 1f,
+			VAlign = 0f,
+			ActiveScale = 1.15f,
+			ActiveRotation = MathHelper.ToRadians(1f)
+		};
+
+		necklace.OnMouseOver += UpdateMouseOver;
+		necklace.OnMouseOut += UpdateMouseOut;
+
+		necklace.Predicate = (item, _) => item.accessory && item.wingSlot <= 0;
+
+		inventoryRoot.Append(necklace);
```

Comment:
Just me being confused - why does the necklace slot have a predicate for being a wing?

Reply:
Oh, for NOT being a wing. Ignore me.